src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -379,9 +379,9 @@ def _get_version_string(name, version_dict):
             else:
                 _print(ext.name.ljust(20) + (ext.version or 'Unknown').rjust(20))
         _print()
-    _print("Python location '{}'".format(sys.executable))
-    _print("Extensions directory '{}'".format(EXTENSIONS_DIR))
     import os
+    _print("Python location '{}'".format(os.path.abspath(sys.executable)))
```

Comment:
Use `abspath` so on Ubuntu the bundle python won't show path like `/usr/bin/../../opt/az/bin/python3`, instead it will just show `/opt/az/bin/python3`.